esrc/rescueoverview.php

```diff
@@ -473,4 +483,86 @@ function displayNotes($row, $isCoord = 0, $isSARAgent = 0)
 		}
 	}
 }
+
+/**
+ * Check killboard and update rescue status if activity detected
+ * @param unknown $data
+ */
+function checkKbActivity($data) {
+
+	// create a new database connection
+	$database = new Database();
+	$kb = new KillboardActivity($database);
+	
+	$lastCheck = strtotime($kb->getLastKillboardCheck());
+	$yesterday = strtotime("-1 day");
+	
+	$isNewCheckRequired = $yesterday > $lastCheck;
+	
+	if (!$isNewCheckRequired) {		
+		return;
+	}
+	
+	$kb->updateLastKillboardCheck();
+
+	$rescue = new Rescue($database);
+	
+	foreach ($data as $row) {
+	
+		// have to wait one day because we don't have access to exact time SAR was opened
+		$since = date("Ymd0000", strtotime($row['lastcontact'] . '+1 day'));
+		$characterId = getCharacterId($row['pilot']);
+		
+		if (isKillboardActive($characterId, $since)) {				
+			$rescue->setStatus($row['id'], 'check-killboard');
```

Comment:
I have not looked into this, but what will adding this new status do to other queries possibly not accounting for this new status?